src/Makefile.am

```diff
@@ -261,6 +261,7 @@ BITCOIN_CORE_H = \
   torcontrol.h \
   txdb.h \
   txmempool.h \
+  txmempool_entry.h \
```

Comment:
If this is moved, why not move it to the right place, that is to `kernel/txmempool_entry.h`?

Reply:
@MarcoFalke #26609.